### DISCUSSION TITLE
No valid Repo

### DISCUSSION THREAD

**Discussion by jab42** [ksheumaker/homeassistant-apsystems_ecur#117] [Q&A] [answered]
hi,
if i want do add the repo, i always get:
https://github.com/ksheumaker/homeassistant-apsystems_ecur is not a valid add-on repository
how may i install?

**Comment by HAEdwin** [comment, marked as answer]
Can't reproduce, what kind of hardware and version are you running HA on? Describe the individual steps you take to install the integration. 
I think this is your issue: https://github.com/ksya/ha-nefiteasy/issues/31
What part of the documentation can I improve to prevent this from happening?

**Reply by jab42** [reply]
maybe you should just give a hint to https://smart-home-assistant.de/home-assistant-hacs-installieren/ :-)
"install repo" is misunderstandable if you dont know hacs.

**Comment by HAEdwin** [comment]
I added the smart-home-assistant.de link and removed "component" in the readme , thanks for the suggestion.
